src/luaconf.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-const llimit = require('./llimit.js');
+const llimit = require('./llimits.js');
```

Comment:
luaconf should not be including llimits.js at all.

Reply:
Yes but luaconf.h uses constants from llimits.h in a macro (that's why it doesn't include it)

Reply:
limits.h, not llimits.h